src/sentry/snuba/sessions_v2.py

```diff
@@ -138,7 +138,7 @@ def extract_from_row(self, row, group):
 
 
 def finite_or_none(val):
-    if not math.isfinite(val):
+    if isinstance(val, (int, float)) and not math.isfinite(val):
```

Comment:
the return value of histogram is an array of arrays. `math.isfinite` will crash on that value